### ISSUE TITLE
[BUG]Spectator mode can't open properly.

### ISSUE THREAD

**Issue by desu7** [AoTTG-2/AoTTG-2#317]
**Describe the bug**
Upon pressing 1 to open the player console, you get a NullReference bug and if you type the /restart command in the chat to restart the level, you can't open the Spawn menu anymore, except if you type /specmode in the chat which "exits" spectator mode and you can open the spawn menu again by pressing 1.

**To Reproduce**
1. Go to any map and game mode.
2. Get killed
3. Press 1, this triggers a NullReference bug in the console. Then if you type /restart in the chat before the automatic restart countdown ends you'll restart the level, sometimes you'll have to press enter twice to send the message.
4. You won't be able to open the Spawn menu by pressing 1 anymore, only way to fix is either by exiting the level or typing /specmode in the chat.

**Expected behavior**
It should always open when pressing '1'.

**Additional context**
The bug won't be triggered as long as you don't press '1' after dying. It won't get triggered if you press '1' twice or more times neither, because once the Spawn menu is opened, it won't close except if choose your side, and you can't type any commands this way.

**Contributor instructions** (Do not modify)
1. Try to recreate the bug via the reported reproduction steps. If the bug can be reproduced, remove the investigation label. If the bug cannot be reproduced, please add a comment, attach "On Hold" label and inform the submitter that the bug cannot be reproduced. If there is no reply after 2 weeks, the issue can be closed.
2. Follow the default development team procedures (Create a new branch, apply your fix, create a pull request, submit new builds for testing team in dev squad)
3. Once testing team has confirmed the issue is resolved and the pull request is accepted, the issue can be closed.

**Comment by Lithrun** [comment]
Might be related to #265. Will forward a build once that one is complete, so you can verify if this was actually resolved in the issue, otherwise we will keep this issue open for additional fixes.
